### PR TITLE
CB-15588 set cache-max-ttl to 30 sec for unbound

### PR DESCRIPTION
**Pull request by sodre90**
Tested on 4 cluster, master node was repaired in 20 mins on all of them.

### CODE REVIEW

**Review by horadla23 — APPROVED** [review]

**Review by biharitomi — APPROVED** [review]

File: orchestrator-salt/src/main/resources/salt/salt/unbound/init.sls [hunk 1]
```diff
@@ -43,6 +43,13 @@ include_access_config:
     - repl: 'include: "/etc/unbound/access.conf"'
     - unless: grep "/etc/unbound/access.conf" /etc/unbound/unbound.conf
 
+set_max_ttl:
+  file.replace:
+    - name: /etc/unbound/unbound.conf
+    - pattern: '# cache-max-ttl:.*'
+    - repl: 'cache-max-ttl: 30'
+    - unless: grep 'cache-max-ttl:\s30'  /etc/unbound/unbound.conf
```
Comment (lacikaaa) [inline orchestrator-salt/src/main/resources/salt/salt/unbound/init.sls:51]: could you drop this? it's unnecessary

Comment (sodre90) [reply orchestrator-salt/src/main/resources/salt/salt/unbound/init.sls:51]: what do you mean on "this" :D

Comment (lacikaaa) [reply orchestrator-salt/src/main/resources/salt/salt/unbound/init.sls:51]: the line I added the comment to, the `unless` part

Comment (sodre90) [reply orchestrator-salt/src/main/resources/salt/salt/unbound/init.sls:51]: 4 line was selected 48-51

File: orchestrator-salt/src/main/resources/salt/salt/unbound/init.sls [hunk 2]
```diff
@@ -43,6 +43,13 @@ include_access_config:
     - repl: 'include: "/etc/unbound/access.conf"'
     - unless: grep "/etc/unbound/access.conf" /etc/unbound/unbound.conf
 
+set_max_ttl:
+  file.replace:
+    - name: /etc/unbound/unbound.conf
+    - pattern: '# cache-max-ttl:.*'
```
Comment (lacikaaa) [inline orchestrator-salt/src/main/resources/salt/salt/unbound/init.sls:49]: if we want to ensure it's always has this value the pattern could look like:
`(# )?cache-max-ttl:.*`

Comment (lacikaaa) [reply orchestrator-salt/src/main/resources/salt/salt/unbound/init.sls:49]: for extra safety also
`- append_if_not_found: True`
could be added :)

**Comment by sodre90** [comment]
@lacikaaa thanks for the helpful comments, I addressed them

**Review by lacikaaa — APPROVED** [review]

**Pull request merged**